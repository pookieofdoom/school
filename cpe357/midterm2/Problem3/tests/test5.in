0 
7    3 5 1 4 7 8 8
0
