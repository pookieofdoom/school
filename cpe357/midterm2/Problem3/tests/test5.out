3, 5, 1, 4, 7, 8, 8
Expected amount of memory used
